src/InfluxDB.LineProtocol/Client/LineProtocolClient.cs

```diff
@@ -36,20 +36,39 @@ public Task<LineProtocolWriteResult> WriteAsync(LineProtocolPayload payload, Can
 
             payload.Format(writer);
 
-            return SendAsync(writer.ToString(), cancellationToken);
+            return SendAsync(writer.ToString(), Precision.Nanoseconds, cancellationToken);
```

Comment:
Small thing, here we've got a variable `writer` that's a `StringWriter`, below we are using `LineProtocolWriter writer` ... my eyes keep wanting to see some symmetry that's not there :-) ... Could we rename this one `stringWriter` or something similar just to break the similarity?